packages/shared/src/hooks/squads/useSquadPostModeration.ts

```diff
@@ -0,0 +1,114 @@
+import { useMutation } from '@tanstack/react-query';
+import { useCallback } from 'react';
+import {
+  PostModerationReason,
+  SquadPostRejectionProps,
+  squadApproveMutation,
+  squadRejectMutation,
+} from '../../graphql/squads';
+import { useLazyModal } from '../useLazyModal';
+import { LazyModal } from '../../components/modals/common/types';
+import { usePrompt } from '../usePrompt';
+import { useToastNotification } from '../useToastNotification';
+
+export const rejectReasons: { value: PostModerationReason; label: string }[] = [
+  {
+    value: PostModerationReason.OffTopic,
+    label: 'Off-topic post unrelated to the squad’s purpose',
```

Comment:
Please make sure we always write `Squad` capitalized :)

Reply:
Right. Should be updated now.